src/main/java/com/greenfiling/smclient/SupplierCostClient.java

```diff
@@ -0,0 +1,62 @@
+package com.greenfiling.smclient;
+
+import com.google.gson.reflect.TypeToken;
+import com.greenfiling.smclient.internal.ApiClient;
+import com.greenfiling.smclient.model.SupplierCost;
+import com.greenfiling.smclient.model.exchange.Index;
+import com.greenfiling.smclient.model.exchange.Show;
+import com.greenfiling.smclient.model.exchange.SupplierCostFilter;
+import com.greenfiling.smclient.model.internal.FilterBase;
+
+public class SupplierCostClient extends ApiClient<SupplierCost, SupplierCost, SupplierCost> {
+
+  /**
+   * consider this to be configurable in the event of multiple cost lookups
+   */
+  public static final String ENDPOINT = "itx/zipcodes";
+
+  private SupplierCostFilter request;
+
+  public SupplierCostClient(ApiHandle handle) {
+    super(handle);
+    setEndpoint(ENDPOINT);
+
+    // @formatter:off
+    setShowType(new TypeToken<Show<SupplierCost>>() {}.getType());
+    setIndexType(new TypeToken<Index<SupplierCost>>() {}.getType());
+    // @formatter:on
+  }
+
+  @Override
```

Comment:
recommend commenting out, see comment on setIndexType above

Reply:
@jetmore  Unless you strongly feel otherwise, I removed any code related to the index implementation and provided a class-level Javadoc comment about it at the top.

Reply:
works for me